controllers/component_build_controller.go

```diff
@@ -110,6 +119,34 @@ func getProvisionTimeMetricsBuckets() []float64 {
 	return []float64{5, 10, 15, 20, 30, 60, 120, 300}
 }
 
+type BuildStatus struct {
+	Simple *SimpleBuildStatus `json:"simple,omitempty"`
+	PaC    *PaCBuildStatus    `json:"pac,omitempty"`
+	// Shows build methods agnostic messages, e.g. invalid build request.
+	Message string `json:"message,omitempty"`
```

Comment:
From the requester point of view, if Message is not empty, it means the request is invalid and cannot be handled by build-service, and requester is able to get the reason provided by build-service from this Message field. Is my understanding correct?

Reply:
This field is used to show error for invalid build requests. Otherwise set to `done`.